WireDomain/Sources/WireDomain/Notifications/Builders/Conversation/ConversationEventNotificationBuilder.swift

```diff
@@ -129,8 +131,12 @@ extension ConversationEventNotificationBuilder {
 
             let eventTimeStamp = time
             let lastReadTimestamp = await conversationLocalStore.lastReadServerTimestamp(conversation)
+            // `selfUser` can be the sender for `mlsMessageAdd` and `proteusMessageAdd` events:
+            // calls can be answered on another device,
+            // so sending notifications is still relevant.
+            let isSenderSelfUserAllowed = isMessageAddEvent ? true : !isSenderSelfUser
```

Comment:
question: why do we do checks in `ConversationEventNotificationBuilder` and not `ConversationCallingEventNotificationBuilder`? I would expect that if we detect a calling event then we handle it the latter and no changes are needed here.

Reply:
we filter our events where selfUser is sender in  `ConversationEventNotificationBuilder`, line #139
validation in the `ConversationCallingEventNotificationBuilder` will be the next step, but we have filter here

Reply:
Could we change the condition to:

```
if isCallMessage {
    // We should try to handle all call messages
    return true
} else if isSenderSelfUser {
    return false
} else if isConversationMuted {
    return false
} else {
    return true
}
```

Reply:
discussed in 1:1